WooCommerce/Classes/ViewRelated/Blaze/BlazeCampaignList/BlazeCampaignListViewModel.swift

```diff
@@ -134,9 +163,16 @@ private extension BlazeCampaignListViewModel {
         resultsController.onDidResetContent = { [weak self] in
```

Comment:
We should rename this into `blazeCampaignResultsController` as we now have another `ResultsController` in this file.

Reply:
The productResultsController is refactored out of the viewmodel so this is resolved.